app/client/src/features/progress/reducer.js

```diff
@@ -13,6 +13,7 @@ const initialState = {
   sections: [
     'templates',
     'profile',
+    'about',
```

Comment:
There is a `Section` type in `client/src/common/types.js` that will need to be updated as well for this change.